lib/ansible/module_utils/network/eric_eccli/eric_eccli.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2019 Ericsson AB.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils._text import to_text
+from ansible.module_utils.basic import env_fallback
+from ansible.module_utils.network.common.utils import to_list, ComplexList
+from ansible.module_utils.connection import Connection, ConnectionError
+
+_DEVICE_CONFIGS = {}
+
+eric_eccli_provider_spec = {
```

Comment:
Provider spec is added for legacy reasons to support connection=local in other network platforms. Since all new ssh based network modules will support connection=network_cli only this dict is not required and can be removed

Reply:
@ganeshrn  Thank you, I will remove it.